src/utils/require-template.js

```diff
@@ -17,7 +17,14 @@ function isValidTemplate(template) {
   );
 }
 
-module.exports = function(template) {
+const getOcTemplate = (path) => {
+  if (require.cache && !!require.cache[path]) {
+    delete require.cache[path];
+  }
+  return require(path);
+};
+
+module.exports = function (template) {
   let ocTemplate;
   const localTemplate = path.join(
```

Comment:
This module https://github.com/substack/node-resolve seems like could be good for the require template functionality, /cc @nickbalestra so that we could change both local and relative templates to look into those paths and find the dependency. This would allow us to have a consistent dev experience between different versions of npm (some flattening the tree, some not) and yarn?